### PR TITLE
Added variables to build Jenkins Docker image from Terraform Jenkins AWS module

### PR DESCRIPTION
**Pull request by ivan-pinatti**
Hi,

This PR referes to the issue [https://github.com/cloudposse/terraform-aws-jenkins/issues/21](https://github.com/cloudposse/terraform-aws-jenkins/issues/21)

I've added variables to build the Jenkins Docker image.

Cheers,

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -113,6 +113,26 @@ resource "aws_codebuild_project" "default" {
       "name"  = "GITHUB_TOKEN"
       "value" = "${var.github_token}"
     }
+
+    environment_variable {
+      "name"  = "JENKINS_URL"
```
Comment (aknysh) [inline main.tf]: this is related to `Jenkins`, it should not be in this module which is supposed to be generic

File: variables.tf [hunk 2]
```diff
@@ -49,6 +49,26 @@ variable "github_token" {
   description = "(Optional) GitHub auth token environment variable (`GITHUB_TOKEN`)"
 }
 
+variable "jenkins_url" {
```
Comment (aknysh) [inline variables.tf]: this is related to Jenkins, it should not be in this module which is supposed to be generic

**Comment by ivan-pinatti** [comment]
@aknysh please refer to [https://github.com/cloudposse/jenkins/pull/11#issuecomment-385780016](https://github.com/cloudposse/jenkins/pull/11#issuecomment-385780016)

**Comment by Jamie-BitFlight** [comment]
@ivan-pinatti I've just done a pull request that will make the environment variables dynamic.
So you can add as many extra ones as you like
https://github.com/cloudposse/terraform-aws-codebuild/pull/16

**Comment by ivan-pinatti** [comment]
Looks promising @Jamie-BitFlight , thanks for it.

**Comment by aknysh** [comment]
@ivan-pinatti 
the issue was solved in https://github.com/cloudposse/terraform-aws-codebuild/releases/tag/0.7.1
it added additional ENV vars

**Pull request closed without merging**